src/main/kotlin/com/openlattice/postgres/RowSecurityPolicyRequest.kt

```diff
@@ -0,0 +1,14 @@
+package com.openlattice.postgres
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.*
+
+data class RowSecurityPolicyRequest(
```

Comment:
Can you please add some javadoc, what the class (and relevant fields) represent?